packages/test-interface-criteria/src/index.js

```diff
@@ -4,12 +4,54 @@
  */
 const criteria = require("./criteria").criteria.map(x => new x());
 
+function getOffers(opportunity) {
+  return opportunity.offers || (opportunity.superEvent && opportunity.superEvent.offers) || []; // Note FacilityUse does not have bookable offers, as it does not allow inheritance
+}
+
+function filterRelevantOffers(criteria, opportunity) {
```

Comment:
I've been quite confused reading these functions because the param name is the same as the module-level variable defined in line 5. Again, this is something ESLint would catch. You could perhaps rename the module-level variable to `allCriteria`?

Reply:
Good idea, yes will sort with ESLint stuff